gse_partner_commission/views/account_move.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_move_form_inherit" model="ir.ui.view">
+        <field name="name">account.move.form.inherit</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button class="oe_stat_button"
+                        name="action_view_purchase_orders"
+                        type="object"
+                        icon="fa-shopping-cart"
+                        attrs="{'invisible': [('purchase_order_count', '=', 0)]}">
+                    <field string="Purchase Orders" name="purchase_order_count" widget="statinfo"/>
+                </button>
+            </xpath>
```

Comment:
Great job adding a new button for the purchase order in the invoice view. However, there is already a button in the invoice view that is visible due to certain conditions. I suggest you use that button instead of creating a new one to avoid duplication. :+1: Make sure it connects to an appropriate action as well. :100: